test/unit/routes/signed-out-routes.test.js

```diff
@@ -0,0 +1,39 @@
+import { vi, beforeEach, describe, test, expect } from 'vitest'
+import { signedOut } from '../../../src/routes/signed-out-routes'
+
+import { signedOutPresenter } from '../../../src/presenters/signed-out-presenter'
+
+const mockView = vi.fn()
+
+const mockH = {
+  view: vi.fn().mockReturnValue(mockView)
+}
+
+vi.mock('../../../src/presenters/signed-out-presenter', () => ({
```

Comment:
```suggestion
// Mocks
vi.mock('../../../src/presenters/signed-out-presenter', () => ({
```

Reply:
We want to keep code comment minimal, so I don't consider the comment line to be necessary.